src/test/java/org/folio/rest/api/DereferencedItemStorageTest.java

```diff
@@ -0,0 +1,245 @@
+    itemToCreate.put("temporaryLocationId", annexLibraryLocationId.toString());
+    itemToCreate.put("_version", 1);
+
+    return itemToCreate;
+  }
+
+  private static JsonObject smallAngryPlanet(UUID itemId, UUID holdingsRecordId) {
+    return createItemRequest(itemId, holdingsRecordId, "036000291452");
+  }
+
+  static JsonObject nod(UUID itemId, UUID holdingsRecordId) {
+    return createItemRequest(itemId, holdingsRecordId, "565578437802");
+  }
+
+  static JsonObject nod(UUID holdingsRecordId) {
+    return nod(UUID.randomUUID(), holdingsRecordId);
+  }
+
+  private static JsonObject uprooted(UUID itemId, UUID holdingsRecordId) {
+    return createItemRequest(itemId, holdingsRecordId, "657670342075");
+  }
+
+  private Response attemptFindByCql(String badSearchQuery) {
+    CompletableFuture<Response> searchCompleted = new CompletableFuture<>();
+
+    client.get(dereferencedItemStorage("?query=") + urlEncode(badSearchQuery),
+      StorageTestSuite.TENANT_ID, ResponseHandler.json(searchCompleted));
+    try{
+      return searchCompleted.get(5, TimeUnit.SECONDS);
+    } catch(Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  private Response attemptFindById(String badId) {
+    CompletableFuture<Response> searchCompleted = new CompletableFuture<>();
+
+    client.get(dereferencedItemStorage("/") + urlEncode(badId),
+      StorageTestSuite.TENANT_ID, ResponseHandler.json(searchCompleted));
+    try{
+      return searchCompleted.get(5, TimeUnit.SECONDS);
+    } catch(Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  private DereferencedItems findByCql(String searchQuery) {
+    CompletableFuture<Response> searchCompleted = new CompletableFuture<>();
+
+    client.get(dereferencedItemStorage("?query=") + urlEncode(searchQuery),
+      StorageTestSuite.TENANT_ID, ResponseHandler.json(searchCompleted));
+    try{
+      return searchCompleted.get(5, TimeUnit.SECONDS).getJson()
+      .mapTo(DereferencedItems.class);
+    } catch(Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  private DereferencedItem findById(String id) {
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+    client.get(dereferencedItemStorage("/" + id), TENANT_ID, json(getCompleted));
+    try {
+      return getCompleted.get(5, SECONDS).getJson()
+      .mapTo(DereferencedItem.class);
+    } catch (InterruptedException | ExecutionException | TimeoutException e) {
+      throw new RuntimeException(e);
+    }
+  }
+  private DereferencedItems getAll() {
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+    client.get(dereferencedItemStorage(""), TENANT_ID, json(getCompleted));
+    try {
+      return getCompleted.get(5, SECONDS).getJson()
+      .mapTo(DereferencedItems.class);
+    } catch (InterruptedException | ExecutionException | TimeoutException e) {
+      throw new RuntimeException(e);
+    }
+  }
```

Comment:
I suggest using sneaky throwing instead of this

(and to provide whitespace between method definitions)

```suggestion

  @SneakyThrows
  private DereferencedItems getAll() {
    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
    client.get(dereferencedItemStorage(""), TENANT_ID, json(getCompleted));

    return getCompleted.get(5, SECONDS).getJson()
      .mapTo(DereferencedItems.class);
  }
```

Reply:
@K-Felk This has been resolved without the code changing, does that mean it's an un-pushed commit or a rejected suggestion?